internal/cmd/test.go

```diff
@@ -165,23 +165,33 @@ func runCMD(mainFunc func(), args []string) (err error) {
 
 func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, commonTestCase CommonCMDTestCase, prefix string) (string, string, int, bool) {
 	stdout, stderr, exitCode, err := RunCMDTest(test, mainFunc, args, commonTestCase.LogLevel)
+
+	logOutputs := func() {
+		test.Logf("Stdout: '%s'.", stdout)
+		test.Logf("Stderr: '%s'.", stderr)
```

Comment:
Change the formatting so it is more clear what is in stdout/stderr.
Like put markers around it and start it on a new line.